day_one/index.js

```diff
@@ -0,0 +1,32 @@
+const testData = require("./test_input.js");
+
+const problemOne = () => {
+  const fuelRequired = testData.map(mass => calculateFuelRequired(mass));
+  return fuelRequired.reduce((prev, cur) => prev + cur);
```

Comment:
this works because javascript will convert undefined and an integer into an integer, but generally you want to have a second argument with reduce as the initial value, so 

.reduce((acc, i) => acc + i, 0)

Reply:
Cool, good to know!